src/entities/mellow-stateless/getters/types.ts

```diff
@@ -0,0 +1,117 @@
+import { BigNumber } from 'ethers';
+
+export type ContractOptimiserInfo = {
+  token: string;
+  feePerDeposit: BigNumber;
+  accumulatedFees: BigNumber;
+  pendingDepositsCount: BigNumber;
+  tokenBalance: BigNumber;
+  ethBalance: BigNumber;
+  isRegisteredForAutoRollover: boolean;
+  erc20RootVaults: {
+    rootVault: string;
+    latestMaturity: BigNumber;
+    vaultCompleted: boolean;
+    vaultPaused: boolean;
+    pendingUserDeposit: BigNumber;
+    committedUserDeposit: BigNumber;
+    canWithdrawOrRollover: boolean;
+  }[];
+};
+
+export type VaultInfo = {
```

Comment:
hmm this can be confused with the type from `mellow-stateless/config/types`

Reply:
https://github.com/Voltz-Protocol/v1-sdk/pull/188/commits/c589f05e4a6a54fd84e3b201d352a2d1316d73f1